src/main/java/com/wdtinc/mapbox_vector_tile/adapt/jts/model/JtsLayer.java

```diff
@@ -32,10 +33,11 @@ public JtsLayer(String name) {
      * @param geometries
      * @throws IllegalArgumentException when {@code name} or {@code geometries} are null
      */
-    public JtsLayer(String name, Collection<Geometry> geometries) {
-        validate(name, geometries);
+    public JtsLayer(String name, Collection<Geometry> geometries, int extent) {
```

Comment:
No on this, it's a breaking API change. Make a separate 3-arg JtsLayer constructor instead of making all clients add a new parameter. The 1-arg constructor can call this() with 2-args. 2-arg can call this() with 3-arg defaults, etc.